lib/models/purchase_params.dart

```diff
@@ -39,7 +42,11 @@ class PurchaseParams {
   /// for more info.
   ///
   /// [promotionalOffer] iOS only. Promotional offer that will be applied to the product.
-  /// Retrieve this offer using [Purchases.getPromotionalOffer].
+  /// Retrieve this offer using [Purchases.getEligibleWinBackOffersForPackage].
+  ///
+  /// [winBackOffer] iOS only. Win-back offer that will be applied to the product.
+  /// Retrieve this offer using [Purchases.getWinBackOffer]. If both a promotional offer
```

Comment:
I think `Purchases.getWinBackOffer` does not exist. Did you mean `Purchases.getEligibleWinBackOffersForPackage`?
```suggestion
  /// Retrieve this offer using [Purchases.getEligibleWinBackOffersForPackage]. If both a promotional offer
```